server/etcdmain/config.go

```diff
@@ -254,6 +254,7 @@ func newConfig() *config {
 	fs.IntVar(&cfg.ec.ExperimentalCompactionBatchLimit, "experimental-compaction-batch-limit", cfg.ec.ExperimentalCompactionBatchLimit, "Sets the maximum revisions deleted in each compaction batch.")
 	fs.DurationVar(&cfg.ec.ExperimentalWatchProgressNotifyInterval, "experimental-watch-progress-notify-interval", cfg.ec.ExperimentalWatchProgressNotifyInterval, "Duration of periodic watch progress notifications.")
 	fs.DurationVar(&cfg.ec.ExperimentalDowngradeCheckTime, "experimental-downgrade-check-time", cfg.ec.ExperimentalDowngradeCheckTime, "Duration of time between two downgrade status check.")
+	fs.DurationVar(&cfg.ec.WarnApplyDuration, "experimental-apply-warning-duration", cfg.ec.WarnApplyDuration, "Time duration after which a warning is generated if request takes more time.")
```

Comment:
"Time duration after which a warning is generated if applying request takes more time."

Reply:
@spzala @jingyih Changes recommended done & modified code committed